src/Sulu/Bundle/TagBundle/Trash/TagTrashItemHandler.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TagBundle\Trash;
+
+use Sulu\Bundle\TagBundle\Admin\TagAdmin;
+use Sulu\Bundle\TagBundle\Tag\TagInterface;
+use Sulu\Bundle\TagBundle\Tag\TagManagerInterface;
+use Sulu\Bundle\TrashBundle\Application\TrashItemHandler\RestoreTrashItemHandlerInterface;
+use Sulu\Bundle\TrashBundle\Application\TrashItemHandler\StoreTrashItemHandlerInterface;
+use Sulu\Bundle\TrashBundle\Domain\Factory\TrashItemFactoryInterface;
+use Sulu\Bundle\TrashBundle\Domain\Model\TrashItemInterface;
+
+class TagTrashItemHandler implements StoreTrashItemHandlerInterface, RestoreTrashItemHandlerInterface
```

Comment:
Would mark this as final as I don't see this as extension point and I think this service never need to be mocked else we should create a `TagTrashItemHandlerInterface`:

```suggestion
final class TagTrashItemHandler implements StoreTrashItemHandlerInterface, RestoreTrashItemHandlerInterface
```